src/main/java/org/ssu/edu/teachua/db/entities/ClubsCategory.java

```diff
@@ -0,0 +1,19 @@
+package org.ssu.edu.teachua.db.entities;
+
+import lombok.Data;
+import org.ssu.edu.teachua.db.annotation.Column;
+import org.ssu.edu.teachua.db.annotation.TableDB;
+
+import java.math.BigInteger;
+
+@TableDB(name = "club_category")
+@Data
+public class ClubsCategory extends Entity {
+
+    @Column(name = "club_id")
+    public BigInteger clubId;
+
+    @Column(name = "category_id")
+    public BigInteger categoryId;
+
+}
```

Comment:
do we need to specify here:
 **@ManyToOne(foreignTable = "clubs", foreignColumnDB = club_id")
    private Club club;**
    or no?